tests/api_tests/users/test_views.py

```diff
@@ -11,14 +10,14 @@
 class TestUsers(ApiTestCase):
 
     def setUp(self):
-        ApiTestCase.setUp(self)
+        super(TestUsers, self).setUp()
```

Comment:
FWIW, I've just added a guideline about calling superclass methods to the docs: https://cosdev.readthedocs.org/en/latest/style_guides/python.html#calling-superclasses-methods